### ISSUE TITLE
Bug: Poor Color Contrast for Searchbox

### ISSUE THREAD

**Issue by arjun44** [SUPERCILEX/gnome-clipboard-history#90]
### Describe the bug

When typing in the search field using the default GNOME shell theme (Adwaita), entered text is white instead of black. Wrong color contrast is also present in other themes.

### How To Reproduce
1. Ensure that you are using the default (Adwaita) GNOME shell theme.
2. Open up the clipboard indicator from the panel.
3. Search for text in the searchbar
4. Notice how the text is the wrong color

![image](https://user-images.githubusercontent.com/41452613/182834099-220ecede-e923-4030-b0a4-0d82c78a8a82.png)
Note: When the mouse is hovering over the text searchbar field there is no shadow, meaning that there is almost 0 contrast.

#### Versions

```
GNOME Shell 42.3.1
GCH   Version: 16
Could not load schemas from /usr/share/gnome-shell/extensions/clipboard-history@alexsaveau.dev/schemas: Failed to open file “/usr/share/gnome-shell/extensions/clipboard-history@alexsaveau.dev/schemas/gschemas.compiled”: open() failed: No such file or directory
```
Note: The extension is installed from a package manager, thus the system-wide filepath. There is no "schemas" directory present.

### Additional context (if a crash, provide stack trace)

N/A

**Comment by SUPERCILEX** [comment]
Can you remove this line and restart gnome (alt F2 then r): https://github.com/SUPERCILEX/gnome-clipboard-history/blob/9a5bdefd240b9a7ff3311bde6a2c8995c26898fa/extension.js#L115

I'm wondering if that might be it.

**Comment by arjun44** [comment]
Yup that seems to have fixed it. Thank you!

**Comment by SUPERCILEX** [comment]
Ok, I'll check to see if it can be removed for the rest of us (that's not a style I've defined).

**Comment by SUPERCILEX** [comment]
Nope, it does affect the styling so I'll leave it. Presumably your gnome theme needs to be updated to use dark text in light theme.
